user-insights.md

```diff
@@ -0,0 +1,20 @@
+# User insights
+
+## Why do we focus on user insights
+
+*We want to make developers awesome!* We believe that offering robust, efficient and capable tools enabling the user to
+create awesome solutions with the Qlik Associative engine will get us there.
+
```

Comment:
This sentence is a bit hard to read..

Reply:
Agree. Maybe:

*We want to make developers awesome!* 

We believe offering robust, efficient and capable data modelling tools will help developers create awesome solutions using the Qlik Associative engine.

Reply:
yepp, that's great!

Reply:
yepp, that's great!

Reply:
yepp, that's great!

Reply:
yepp, that's great!

Reply:
yepp, that's great!